dinner-party.rb

```diff
@@ -0,0 +1,41 @@
+food=["chicken", "pasta", "steak", "fish", "salad"]
+beverages =["soda", "juice", "water", "coffee", "tea"]
+people =["Leshoya","Daniel", "Chris", "Karen", "Tammy"]
+seat_number=["seat 1","seat 2","seat 3","seat 4","seat 5"]
```

Comment:
Generally all assignments to variables have spaces before and after them, this is one of the things Code Climate complained about that I would fix. It should be:
```rb
seat_number = ["seat 1","seat 2","seat 3","seat 4","seat 5"]
```

This applies to the other uses of `=` in here.